src/main/java/com/epam/bookstoreservice/model/UserModel.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.bookstoreservice.model;
+
+import com.epam.bookstoreservice.dto.response.UserResponseDTO;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * the model for userResponseDTO
+ */
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class UserModel extends RepresentationModel<UserModel> {
```

Comment:
You could move this model to a new package called 'hateoas/model' since this feature is related to it.